target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -129,6 +129,11 @@ comfast,cf-e314n-v2)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "green:rssimediumhigh" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "green:rssihigh" "wlan0" "76" "100"
 	;;
+comfast,cf-e355-v2)
+	ucidef_set_led_netdev "lan" "LAN" "cf-e355ac-v2:green:lan" "eth0"
+	ucidef_set_led_wlan "wlan2g" "WLAN2G" "blue:wlan2g" "phy1tpt"
+	ucidef_set_led_wlan "wlan5g" "WLAN5G" "red:wlan5g" "phy0tpt"
```

Comment:
WLAN definitions can be dropped here since you have the triggers DTS.

Reply:
Thanks, fixed in 244749b